packages/apps/hr/src/components/molecules/TimerDeclareButton/TimerDeclareButton.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {useTranslator} from '@axelor/aos-mobile-core';
+import {Button} from '@axelor/aos-mobile-ui';
+
+const TimerDeclareButton = () => {
+  const I18n = useTranslator();
+
+  return (
+    <Button
+      iconName="calendar-check"
+      title={I18n.t('Base_Declare')}
+      onPress={() => console.log('Declare button pressed.')}
```

Comment:
maybe needed to be transorm in js file in future if needed dispatch

Reply:
I will see this later as I may use props for onPress, so I can define the logic in `TimerListScreen.js` (like it is in `ExpenseLinesListScreen.js`)